diff -burN renattach-1.2.2.orig/src/Makefile.am renattach-1.2.2/src/Makefile.am
--- renattach-1.2.2.orig/src/Makefile.am	2005-01-09 19:51:57.986974464 +0100
+++ renattach-1.2.2/src/Makefile.am	2005-01-10 10:51:59.771046048 +0100
@@ -1,5 +1,5 @@
 AM_CFLAGS = -pedantic -Wall -DCONF_DIR=\"$(sysconfdir)\"
 bin_PROGRAMS = renattach
 dist_man_MANS = renattach.1
-renattach_SOURCES = renattach.c settings.c utility.c getopt.c getopt1.c \
+renattach_SOURCES = renattach.c settings.c utility.c \
 	renattach.h strings-en.h utility.h getopt.h
diff -burN renattach-1.2.2.orig/src/renattach.c renattach-1.2.2/src/renattach.c
--- renattach-1.2.2.orig/src/renattach.c	2005-01-09 19:51:57.987974312 +0100
+++ renattach-1.2.2/src/renattach.c	2005-01-10 10:52:10.653391680 +0100
@@ -26,7 +26,7 @@
 #include "renattach.h"
 #include "strings-en.h"
 #include "utility.h"
-#include "getopt.h"
+#include <getopt.h>
 #include <memory.h>
 #include <stdio.h>
 #include <stdlib.h>
